src/app/forgot/page.jsx

```diff
@@ -0,0 +1,100 @@
+'use client'
+import { Button, Snackbar, Stack, Typography } from '@mui/material'
```

Comment:
```suggestion
import { Button, Snackbar, Stack, Typography as T } from '@mui/material'
```
No es más cómodo y limpo usar Typography así?